### PR TITLE
Add Rails 6 to appraisals, update spec coverage

### PR DESCRIPTION
**Pull request by smudge**
👋 Howdy!

There were a couple small changes to Rails 6.0 model structure, which should be accounted for in the new 6_0 files in `spec/support/active_record_data`. Aside from that, the appraisal changes should be pretty straightforward.

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 appraise 'rails-5.0' do
   gem 'rails', '>= 5.0.0', '< 5.1'
+  gem 'sqlite3', '< 1.4'
```
Comment (smudge) [inline Appraisals]: Rails 5.0 breaks on sqlite3 1.4 (recently released)

File: .travis.yml [hunk 2]
```diff
@@ -16,16 +16,19 @@ gemfile:
...
+  exclude:
+    - gemfile: gemfiles/rails_6.0.gemfile
+      rvm: 2.3
+    - gemfile: gemfiles/rails_6.0.gemfile
+      rvm: 2.4
+  allow_failures:
     - rvm: ruby-head
-      env: RUBYOPT="--enable-frozen-string-literal"
-  # allow_failures:
-  #   - rvm: ruby-head
```
Comment (smudge) [inline .travis.yml]: From what I can tell, frozen string literals are now the default on ruby-head, which breaks external dependencies.

Comment (smudge) [reply .travis.yml]: (see also: https://github.com/awesome-print/awesome_print/pull/369#issuecomment-527994310)

File: Appraisals (hunk 1 above)
Comment (smudge) [reply Appraisals]: This is similar to #369 but restricts the version pinning to the Rails 5.0 build only.

**Review by BryanH — APPROVED** [review]
This is passing Rails 5

**Comment by BryanH** [comment]
Thank you very much for your contributions! There were a number of other PRs that resolved the above which have been merged. If I missed any, would you mind please submitting a follow-up PR?

**Pull request closed without merging**